fud2/scripts/axi.rhai

```diff
@@ -23,15 +23,25 @@ fn wrapper_setup(e) {
     // For now point to standalone axi-generator.py. Can maybe turn this into a rsrc file?
     let dynamic =
         e.config_constrained_or("dynamic", ["true", "false"], "false");
+    
     let generator_path = if dynamic == "true" {
         "$calyx-base/yxi/axi-calyx/dynamic_axi_generator.py"
     } else {
         "$calyx-base/yxi/axi-calyx/axi_generator.py"
     };
+    
+    let for_xilinx = e.config_constrained_or("xilinx.controlled", ["true", "false"], "false");
+    
+    if for_xilinx == "true" {
+        e.var_("python_args", "--generate-for-xilinx");
```

Comment:
This flag controls whether or not the [`s_axi_control`](https://docs.amd.com/r/en-US/ug1701-vitis-accelerated-embedded/RTL-Kernel-Interface-Requirements) axi4-lite subordinate is generated in the calyx-axi-wrapper.

My thinking was that, `cocotb` can (must?) test functionality of a wrapped program without a `s_axi_control` subordinate. When emulating/execting on hardware via XRT, we need the subordinate.

I realize this adds _another_ permutation of config variables to keep track of, so perhaps we should we only support XRT-compliant axi wrappers for now.